docker/docker_official_images/3.7.0/jvm/Dockerfile

```diff
@@ -78,7 +78,7 @@ RUN set -eux ; \
     chmod -R ug+w /etc/kafka /var/lib/kafka /etc/kafka/secrets; \
     cp /opt/kafka/config/log4j.properties /etc/kafka/docker/log4j.properties; \
     cp /opt/kafka/config/tools-log4j.properties /etc/kafka/docker/tools-log4j.properties; \
-    cp /opt/kafka/config/kraft/reconfig-server.properties /etc/kafka/docker/server.properties; \
+    cp /opt/kafka/config/kraft/server.properties /etc/kafka/docker/server.properties; \
```

Comment:
We need to remove `kraft` from the path here

Reply:
I shouldn't change this one. This dockfile somehow is only for AK 3.7. 
We should revert the change. WDYT ?

Reply:
Glad it was a lucky guess that turned out to be the right fix!  😅 
I just checked the branch 3.7 and I understand now.

Reply:
Hi @TaiJuWu 
I will file a jira to raise a discussion about it.